Sources/Basics/Triple.swift

```diff
@@ -259,6 +262,11 @@ extension Triple {
             return ".dll"
         case .wasi:
             return ".wasm"
+        case .noneOS:
+            // Better as dynamic error or "fatalError", but it is a larger
+            // undertaking to teach spm that dylibs are not available on some
+            // platforms.
+            return ".os-none-dynamic-library"
```

Comment:
I personally would prefer `fatalError` here

Reply:
IIRC, we had problems with that in the WASI case? I don't remember if they were specific to that or a workaround for dylib extensions to be eagerly asked for by SwiftPM.

Reply:
WASI just has static stdlib linking and everywhere else hardcoded to be enabled by default AFAIR

Reply:
Changed to fatal error, but I had to make the use of this string lazy to avoid crashing in normal use (for triple `armv7em-apple-none-macho`)